solver/src/main.rs

```diff
@@ -200,12 +200,11 @@ async fn main() {
 
     let pool_aggregator = PoolAggregator::from_providers(&pair_providers, &web3).await;
 
-    // TODO - use Filtered-Cached PoolFetchers here too.
```

Comment:
Are we removing this because its no longer necessary? Or were you going to introduce it here?

Reply:
As I am saying in the commit message I don't think the filtering part ever made sense. But I guess we might still want to cache this. I'll add that part of the comment back.